inputfiles/overridingTypes.json

```diff
@@ -2566,6 +2566,33 @@
                         }
                     }
                 }
+            },
+            "CompileError": {
+                "extends": "Error",
+                "constructor": {
+                    "override-signatures": [
+                        "new(message?: string): CompileError",
+                        "(message?: string): CompileError"
+                    ]
+                }
+            },
+            "LinkError": {
+                "extends": "Error",
+                "constructor": {
+                    "override-signatures": [
+                        "new(message?: string): LinkError",
+                        "(message?: string): LinkError"
+                    ]
+                }
+            },
+            "RuntimeError": {
+                "extends": "Error",
+                "constructor": {
+                    "override-signatures": [
+                        "new(message?: string): RuntimeError",
+                        "(message?: string): RuntimeError"
+                    ]
+                }
             }
```

Comment:
These types are actually defined using **ECMA‑262**’s <code><var>NativeError</var></code> object structure:
- <https://webassembly.github.io/spec/js-api/#error-objects>
- <https://tc39.es/ecma262/#sec-nativeerror-object-structure>

Reply:
Extracted to: <https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/962>